### PR TITLE
File picker

### PR DESCRIPTION
**Pull request by paul-nameless**
#125

### CODE REVIEW

**Comment by paul-nameless** [comment]
@lightme16 what do you think about this patch? Is it usable?
I have only one concern. What if you want to choose file (pic for example) and send it in full resolution as document instead of pic (same for video, telegram decreases quality).
I don't want to add extra key presses or dialogs which ask but this is something that can be useful and also this is what official client does, allows to choose between document and photo.

File: tg/controllers.py [hunk 1]
```diff
@@ -320,6 +309,30 @@ def delete_msgs(self) -> None:
             return
         self.present_info("Message deleted")
 
+    @bind(msg_handler, ["S"])
+    def choose_and_send_file(self) -> None:
```
Comment (lightme16) [inline tg/controllers.py:313]: plz add doc string so it will be displayed in help `?`

File: tg/controllers.py [hunk 2]
```diff
@@ -332,6 +345,21 @@ def send_picture(self) -> None:
     def send_audio(self) -> None:
         self.send_file(self.tg.send_audio)
 
+    @bind(msg_handler, ["sv"])
+    def send_video(self) -> None:
```
Comment (lightme16) [inline tg/controllers.py:349]: add doc string

File: tg/controllers.py [hunk 3]
```diff
@@ -320,6 +309,38 @@ def delete_msgs(self) -> None:
...
+        if not file_path or not os.path.isfile(file_path):
+            return self.present_error("No file was selected")
+        mime_map = {
+            "image": self.tg.send_photo,
+            "audio": self.tg.send_audio,
+            "video": self._send_video,
+        }
+        mime = get_mime(file_path)
+        if mime in ("image", "video"):
+            resp = self.view.status.get_input(
```
Comment (lightme16) [inline tg/controllers.py:335]: we need to clean up status line after prompting user.

File: tg/controllers.py [hunk 4]
```diff
@@ -663,8 +668,14 @@ def render(self) -> None:
         self.queue.put(self._render)
 
     def _render(self) -> None:
-        self.render_chats()
-        self.render_msgs()
+        self._render_chats()
+        self._render_msgs()
+        self._render_status()
```
Comment (lightme16) [inline tg/controllers.py:673]: that is incorrect. it creates unexpected behavior when sending msgs with `i`. each render invocation removes msg entered. 
so, status line shouldn't be updated during msg inserting.

**Pull request merged**